spec/controllers/presentations_controller_spec.rb

```diff
@@ -90,6 +90,17 @@
         end
       end
     end
+
+    describe 'DELETE #destroy' do
+      let!(:presentation) { FactoryGirl.create(:presentation) }
```

Comment:
@brendandeere The way this test is implemented it doesn't work without the bang because when presentation gets called in the subject block it immediately destroys the presentation, and so the test to check if the presentation count has changed will fail. Is there a better approach overall for this test?